app-next/components/Volunteer/Volunteer.jsx

```diff
@@ -21,16 +21,12 @@ const Volunteer = ({ volunteer }) => {
 
       <div className={styles.content}>
         <h3 className={styles.name}>{volunteer.name}</h3>
-        <p className={styles.address}>{volunteer.address}</p>
-        <p className={styles.services}>
-          Services:{" "}
-          {volunteer.services?.length
-            ? volunteer.services.join(", ")
-            : "Not specified"}
-        </p>
-        <p className={styles.availability}>
-          Availability: {volunteer.availability || "Not specified"}
-        </p>
```

Comment:
We agreed not to change how components are looking on this page, only fill it with data